Dockerfile.ubuntu

```diff
@@ -0,0 +1,7 @@
+FROM python:3.10.6-slim
```

Comment:
Using the Docker community image provided by Nvidia will achieve better performance and may be worth considering